contracts/Interfaces/Types.sol

```diff
@@ -2,23 +2,18 @@
 pragma solidity ^0.8.0;
 
 import "../lib/LibPerpetuals.sol";
+import "../lib/LibPrices.sol";
 
 interface Types {
     struct FundingRate {
-        uint256 recordTime;
-        uint256 recordPrice;
-        int256 fundingRate; //positive value = longs pay shorts
-        int256 fundingRateValue; //previous rate + (time diff * price * rate)
+        uint256 timestamp;
+        int256 fundingRate;
+        int256 cumulativeFundingRate;
     }
 
-    struct HourlyPrices {
-        uint256 totalPrice;
-        uint256 numTrades;
-    }
-
-    struct PricingMetrics {
-        Types.HourlyPrices[24] hourlyTracerPrices;
-        Types.HourlyPrices[24] hourlyOraclePrices;
+    struct PricesMetrics {
```

Comment:
Nitpick: Why is `PriceMetrics` in here and not in `LibPricing.sol`?

Reply:
Also, is `struct PriceMetrics` even being used anymore?

Reply:
Shouldn't even exist; good catch.

Reply:
Consult cb37d4f